### PR TITLE
Updated quest and created tests for the Forge quest

### PR DESCRIPTION
**Pull request by greglarkin**
LEARNVM-500 #resolved #time 3h

### CODE REVIEW

**Comment by greglarkin** [comment]
@kjhenner Ready for review at your convenience

File: quests/the_forge.md [hunk 1]
```diff
@@ -250,48 +258,52 @@ class pasture (
...
 ---
-  :default_character: <%= $default_character %>
-  :default_message: <%= $default_message %>
-  <% if $db { -%>
-  :db: <%= $db %>
-  <% } -%>
-  :sinatra_settings:
-    :port:   <%= $port %>
-    :server: <%= $sinatra_server %>
+:default_character: <%= $default_character %>
```
Comment (samuelson) [inline quests/the_forge.md]: You still need the indent here

File: quests/the_forge.md [hunk 2]
```diff
@@ -250,48 +258,52 @@ class pasture (
...
+:default_message: <%= $default_message %>
+<%- if $db { -%>
```
Comment (samuelson) [inline quests/the_forge.md]: These are the only lines that need to not be indented, but I don't think you want the <%-. I'm not sure.  I fixed it by just unindenting these lines

Comment (kjhenner) [inline quests/the_forge.md]: If you have a -%> without a <%-, it removes the newline character at the end of that line, but leaves any preceding whitespace (i.e. the indentation), so the next line ends up getting dropped in front of that indentation.

**Pull request merged**